frontend/public/module/k8s/probe.ts

```diff
@@ -13,15 +14,15 @@ import {
 const HookAction = Object.freeze({
   exec: {
     id: 'exec',
-    label: 'Exec Command',
+    label: i18next.t('public~Exec command'),
```

Comment:
These need to be inside a function. `i18next` might not be initialized at this point, and the translations will never be reevaluated. We'll need a function like `getHookActions` instead of a top-level object.